### PR TITLE
[FEAT] generate job events

### PR DESCRIPTION
**Pull request by sindhu-106**
Signed-off-by: sindhuja <sindhuja.gajula@intel.com>

### CODE REVIEW

File: config-files/EventSim.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "eventsimConfig" : {
       "SensorMetadata": "/resources/ForeignSensorMetaData.json",
       "RASMetadata": "/resources/ForeignEventMetaData.json",
+      "JobsMetadata" : "/resources/ForeignJobsMetaData.json",
```
Comment (nbwhite) [inline config-files/EventSim.json:5]: Can you make this change in the docker config as well?

Comment (sindhu-106) [reply config-files/EventSim.json:5]: sure nathan

Comment (sindhu-106) [reply config-files/EventSim.json:5]: > Can you make this change in the docker config as well?

Changes are done.

Comment (nbwhite) [reply config-files/EventSim.json:5]: Thanks!

**Pull request merged**